lib/fireblocks/version.rb

```diff
@@ -2,7 +2,7 @@
 
 module Fireblocks
   MAJOR = 0
-  MINOR = 2
-  TINY = 15
+  MINOR = 3
+  TINY = 0
```

Comment:
@Marie-Burns22 - for my own curiosity, where are you getting these values?

Reply:
Grant suggested that I bump to minor 3 because of the significance of the changes that are not breaking. We did not know why it is called minor and tiny.

Reply:
[Typical software versioning is Major, Minor, Patch](https://en.wikipedia.org/wiki/Software_versioning). So `TINY` is really `Patch`